lib/src/components/NxToast/__tests__/NxToast.test.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React from 'react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import { fireEvent, screen } from '@testing-library/react';
+
+import userEvent from '@testing-library/user-event';
+
+import NxToast from '../NxToast';
+import { NxToastProps } from '../types';
+import { NxErrorAlert } from '../../NxAlert/NxAlert';
+
+describe('NxToast', function() {
+  const simpleProps: NxToastProps = {
+    onClose: () => {},
+    children: <NxErrorAlert>Toast Message</NxErrorAlert>
+  };
+
+  const quickRender = rtlRender(NxToast, simpleProps),
+      renderEl = rtlRenderElement(NxToast, simpleProps);
+
+  it('renders a div with a class .nx-toast', function() {
+    const component = quickRender();
+    const toast = component.container.children[0];
+
+    expect(toast.nodeName).toEqual('DIV');
+    expect(toast).toHaveClass('nx-toast');
+  });
+
+  it('contains an NxErrorAlert with the NxToast message', function() {
+    quickRender();
+
+    const alert = screen.getByRole('alert');
+
+    expect(alert).toBeInTheDocument();
+    expect(alert).toHaveTextContent('Toast Message');
+  });
+
+  it('adds a class specified with the className prop', function() {
+    const component = renderEl({ className: 'foo' });
+
+    expect(component).toHaveClass('foo');
+    expect(component).toHaveClass('nx-toast');
```

Comment:
See how I did this in `NxDescriptionList.test.tsx`.  I tried to keep the test from worrying about the implementation details of which classes are applied by default, while still testing that those same classes are applied when extra classes are also specified

Reply:
> See how I did this in `NxDescriptionList.test.tsx`. I tried to keep the test from worrying about the implementation details of which classes are applied by default, while still testing that those same classes are applied when extra classes are also specified

Thanks for pointing me in the right direction here. Since NxToast is now accepting other attributes as well, I also added in/tested for the id and language attributes. [75abd56](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/75abd5697e903458941a7c13e4ffe9fc2d4b330c)